### PR TITLE
Bootstrap Assignment1 Completed

### PR DESCRIPTION
**Pull request by Prashant698**

### CODE REVIEW

File: prashant/Bootstrap/Assignment1.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    </div>
+                    <div class="col-md-9" >
+                        <div class="card" style="box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);" > 
+                           <div class="float-left"> <i class="fa fa-user-circle-o" style="font-size:36px; padding-left: 20px; margin-top: 23px;"></i> </div>
+                           <div class="card-body" style="float: right; margin-left: 60px; margin-top: -50px;">Snehal Ghodmare Announce something to your class</div>
+                           <div class="float-right"><i class="fa fa-retweet" style="float: right; font-size: 30px; color: gray; margin-top: -55px; margin-right: 30px;"></i></div>  
+                        </div>  
+                        <div class="card" style="margin-top: 20px;">  
+                            <div class="float-left"> <i class="fa fa-clipboard" style="font-size:36px; padding-left: 20px; margin-top: 23px;"></i> </div>
+
```
Comment (sourabh-kanwade) [inline prashant/Bootstrap/Assignment1.html]: Remove Space

File: prashant/Bootstrap/Assignment1.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                            <div class="float-left"> <i class="fa fa-clipboard" style="font-size:36px; padding-left: 20px; margin-top: 23px;"></i> </div>
+                            <div class="card-body" style="float: right; margin-left: 60px; margin-top: -50px;">Snehal Ghodmare posted a new assignment : Evaluation 26 May 2021</div>
+                            <div class="float-right"><i class="fa fa-ellipsis-v" style="float: right; font-size: 30px; color: gray; margin-top: -55px; margin-right: 30px;"></i></div>  
+                         </div> 
+                    </div>
+                </div>       
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (sourabh-kanwade) [inline prashant/Bootstrap/Assignment1.html]: Add new line after line 55

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]

**Review by sourabh-kanwade — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

File: prashant/Bootstrap/Assignment1.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<html lang="en">
+    <head>
+    <title>Bootstrap Card</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (Pawan1005) [inline prashant/Bootstrap/Assignment1.html]: make correct indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: prashant/Bootstrap/Assignment1.html (hunk 1 above)
Comment (Prashant698) [reply prashant/Bootstrap/Assignment1.html]: done

File: prashant/Bootstrap/Assignment1.html (hunk 2 above)
Comment (Prashant698) [reply prashant/Bootstrap/Assignment1.html]: done

File: prashant/Bootstrap/Assignment1.html (hunk 3 above)
Comment (Prashant698) [reply prashant/Bootstrap/Assignment1.html]: done it sir.

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**